src/main/java/hudson/plugins/sidebar_link/LinkAction.java

```diff
@@ -65,7 +65,7 @@ public LinkAction(String urlName, String displayName, String iconFileName) throw
         }
 
         if(StringUtils.isBlank(iconFileName)) {
-            iconFileName = "static/efbf17e4/images/16x16/help.png";
+            iconFileName = "static/79e6e80a/images/svgs/help.svg";
```

Comment:
```suggestion
            iconFileName = "icon-help icon-md";
```
The more future proof solution would be to work with the API and call the icon class instead of relying on a path again.

Reply:
It's not exactly the same icon but I'm fine with it.